duckduckgo/Android
# move night-colors into common-ui module

Task/Issue URL: https://app.asana.com/0/414730916066338/1200721266093175/f
Tech Design URL: 
CC: 

**Description**:
This PR just moves that night/colors file into common-ui according to our guidelines.

**Steps to test this PR**:
1. Add new favorites widget to your home screen using “system default” theme
1. Ensure system theme is honored when changing between light and dark


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
